NAMESPACE

```diff
@@ -7,4 +7,5 @@ importFrom(jsonlite,read_json)
 importFrom(magrittr,"%>%")
 importFrom(rlang,"!!!")
 importFrom(rlang,"%||%")
+importFrom(rlang,`%||%`)
```

Comment:
Is this line duplicative of the previous?  (See [comment](https://github.com/ropensci/riem/pull/48/files#r1892782003) below.)  I'm uncertain about the nuance of backticks vs. double-quotes.